app/views/proposals/_proposal.html.erb

```diff
@@ -86,6 +86,27 @@
                             <% end %>
                           </div>
                         </div>
+                        </div>
+                        <div>
+                          <%= label_tag :assigned_location_id %>
+                        </div>
+                        <div class="mb-3">
+                          <%= f.text_field :assigned_location_id, class: 'form-control w-25', disabled: true, value:@proposal.assigned_location&.code %>
+                        </div>
+                         <div>
+                          <%= label_tag :assigned_date %>
+                        </div>
+                        <div class="mb-3">
+                          <%= f.text_field :assigned_date, class: 'form-control w-25', disabled: true, value:@proposal.assigned_date %>
+                        </div>
+                        <div class="mb-3">
+                          <label for="same_week_as">Same week as: </label>
+                          <label for="same_week_as">(Links to another proposal that is to be scheduled the same week as this proposal.)</label>
+                        </div>
+                         <div class="mb-3">
+                          <label for="week_after">Week after: </label>
+                          <label for="week_after">(Links to another proposal, schedule this proposal the week after the linked proposal)</label>
+                        </div>
 
                         <div class="mb-3">
```

Comment:
The same_week_as and week_after fields are missing - should be a list of proposal codes of the proposals that were accepted by the peer review process for the given year. Status: decision approved ? (ask PM)